app/routes/admin/messages/list.js

```diff
@@ -7,9 +7,9 @@ export default class extends Route.extend(EmberTableRouteMixin) {
     const searchField = 'recipient';
     const filterOptions = this.applySearchFilters([], params, searchField);
     let queryString = {
-      filter        : filterOptions,
-      'page[size]'  : params.per_page || 100,
-      'page[number' : params.page || 1
+      filter         : filterOptions,
+      'page[size]'   : params.per_page || 100,
+      'page[number]' : params.page || 1
     };
     queryString = this.applySortFilters(queryString, params);
     return this.asArray(this.modelFor('admin.messages', queryString));
```

Comment:
Actually the queryString should be in 'admin.messeges' with page[size] and page[number]. But how to pass it there?